docs/docsource/installation/install/install_xdm.rst

```diff
@@ -150,11 +150,11 @@ Deploying XDM
       }
       EOF
 
-#. Create a yaml file for the new XDM's version:
+#. Create a yaml file for the new XDM version:
 
    .. code::
 
-      kubectl apply --namespace zenko -f /srv/scality/zenko-2.0.0-alpha.3/zenkoversion.yaml
+      kubectl apply --namespace zenko -f /srv/scality/zenko-2.0.0-beta.3/zenkoversion.yaml
```

Comment:
I recommend inserting a |version| variable and setting this code sample (and subsequent ones containing a Zenko version number) as parsed literals.

Reply:
I've added a variable for Zenko, but the problem remains for MetalK8s. 

We are recommended to avoid writing the version number in the doc, so should I write "{{version}}" instead of MetalK8s' "2.6.0" version number?

Reply:
```
.. parsed-literal::

   kubectl apply --namespace zenko -f /srv/scality/zenko-|version|/zenkoversion.yaml
```